docs/api/users/delete.md

```diff
@@ -1,7 +1,87 @@
 ---
 id: delete
-title: Delete
+title: Delete user
 sidebar_label: Delete
+keywords:
+  - OpenHIM
+  - API
+  - User
+  - Delete
+description: Delete an OpenHIM User via the API
 ---
 
-Coming soon ...
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+To delete a user record, you will need to make a TLS request to the OpenHIM API for the below method and endpoint.
+
```

Comment:
We are missing a heading here. This heading creates a link in the right navigation to easily jump to a section.